### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by youngzw** [FeiSun/BERT4Rec#14]
I modified `lr=1e-3` (`1e-4` in your paper) and found that the convergence speed is much faster. Similar phenomena also appeared in other code reproduction repositories, e.g, [cr_1](https://github.com/Qdriving/Bert4Rec_Paddle2.0) and [cr_2](https://github.com/jaywonchung/BERT4Rec-VAE-Pytorch).
